examples/transmitter/swagger_server/business_logic/generate_event.py

```diff
@@ -0,0 +1,106 @@
+# Copyright (c) 2021 Cisco Systems, Inc. and its affiliates
+# All rights reserved.
+# Use of this source code is governed by a BSD 3-Clause License
+# that can be found in the LICENSE file.
+
+import time
+from typing import Dict
+
+from swagger_server.events import (
+    Events, SecurityEvent,
+    SessionRevoked,TokenClaimsChange, CredentialChange,
+    AssuranceLevelChange, DeviceComplianceChange
+)
+
+from swagger_server.models import Subject
+
+# TODO: 1. Support RISC
+class GenerateEvent:
```

Comment:
Instead of doing any of this, what if you just set default values on the models in `events.py`?

Reply:
I've set all the values except the subject as default.
Still need these methods to set subject tho,
We can discuss this